src/model/SubaddressAccountModel.h

```diff
@@ -0,0 +1,36 @@
+#ifndef SUBADDRESSACCOUNTMODEL_H
+#define SUBADDRESSACCOUNTMODEL_H
+
+#include <QAbstractListModel>
+
+class SubaddressAccount;
+
+class SubaddressAccountModel : public QAbstractListModel
+{
+    Q_OBJECT
+
+public:
+    enum SubaddressAccountRowRole {
+        SubaddressAccountRole = Qt::UserRole + 1, // for the SubaddressAccountRow object;
```

Comment:
This one is not handled in roleNames and data. Is it a placeholder only?

Reply:
OK, removed